torch/distributed/fsdp/_init_utils.py

```diff
@@ -359,8 +361,12 @@ def _init_param_handle_from_params(
     state._handles.append(handle)
     for module in handle.flat_param._modules:
         state._module_to_handles[module].append(handle)
-    for module in handle.flat_param._root_modules:
```

Comment:
Is this multiple root module case what you mentioned before where we have `parent(child1, child2)` and all parameters are on child1 and child2?

Reply:
Yup.